.github/workflows/scorecards.yml

```diff
@@ -5,7 +5,9 @@ on:
     # Weekly on Saturdays.
     - cron: '30 1 * * 6'
   push:
-    branches: [ main ]
+    branches:
+      - main
+      - staging
```

Comment:
Do we know if OSSF scorecard action allows for non default branch to run?

Reply:
From the scorecard-action [page](https://github.com/ossf/scorecard-action), push should be supported. 
![image](https://github.com/deislabs/ratify/assets/6919333/e23c761c-4c8d-41cb-8ccb-8c8e43629af0)